cli/src/nonce.rs

```diff
@@ -95,16 +95,16 @@ impl NonceSubCommands for App<'_, '_> {
                 .arg(
                     Arg::with_name("nonce_account_keypair")
                         .index(1)
-                        .value_name("NONCE_ACCOUNT")
+                        .value_name("KEYPAIR")
                         .takes_value(true)
                         .required(true)
-                        .validator(is_pubkey_or_keypair)
+                        .validator(is_valid_signer)
```

Comment:
@t-nelson, is that right?

Reply:
This is one field that will be addressed as as part of #8852 ; probably easiest for me if you leave this as-is for now (but it will be updated to `is_valid_signer`, yes).

Reply:
So I'm thinking that either this is the right place for this change or that `--nonce-account-keypair` should be `--nonce-account-pubkey`.